keyboards/atreus/keymaps/nojjan/config.h

```diff
@@ -0,0 +1,15 @@
+#ifndef CONFIG_USER_H
```

Comment:
Could you replace the first three lines (ifndef to include) with just `#pragma once`, and remove the endif at the end?